### ISSUE TITLE
bug(Select): Missing `role` attribute in svg element

### ISSUE THREAD

**Issue by tarlepp** [angular/components#26547]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

When using `<mat-select>` on some page and check it with [ARC Toolkit](https://chrome.google.com/webstore/detail/arc-toolkit/chdkkkccnlfncngelccgbgfmjebmkmce?hl=en), it reports:
```
The <svg> acts as an image, but is missing an explicit role="img".
```

Code for that is there - https://github.com/angular/components/blob/bf6f38abd7b8f115413aa9de08b632b017827ba9/src/material/select/select.html#L27

### Reproduction

Steps to reproduce:
1. Create application that uses `mat-select` component 
2. Run [ARC Tookkit](https://chrome.google.com/webstore/detail/arc-toolkit/chdkkkccnlfncngelccgbgfmjebmkmce?hl=en) for that site
3. You should see errors like:
![image](https://user-images.githubusercontent.com/595561/216240094-74a86b05-3ff5-44f6-9d8b-d526a174f374.png)

### Expected Behavior

Those `svg` elements should have `role` attribute - or those `svg` elements should be hidden.

### Actual Behavior

Those `svg` elements doesn't have `role` attribute.

### Environment

- Angular: 15.1.2
- CDK/Material: 15.1.2
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

**Comment by SaishNarvekar** [comment]
Hi @zarend, 
is this bug still open to work on? i can see similar fix that was given in #27116

**Comment by zarend** [comment]
Hello @SaishNarvekar , looks like #27116  has already fixed this issue.

We welcome contributors. If you're looking for something to do, here are some suggestions for new contributors:

 - Make corrections and maintain our documentation
 - Pick up an issue with label "good first issue". Looks like we have about 16 open right now https://github.com/angular/components/labels/good%20first%20issue
 - Test the components for bugs and file an issue report
 - participate in discussions/stackoverflow/etc.
 
 
-Zach
